lib/features/reps_videos/reps_videos.features.field_base.inc

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
The body field is already coming from a feature of the platform
You don't need to re-export it in field base. This will generate conflict between feature

Reply:
@libetho Indeed, body field removed, thx you for the review!